migrations/20210928081959_tutorial.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table('psychologists', (table) => {
+    table.boolean('hasSeenTutorial');
```

Comment:
We can set default to false explicitely and put it not null.
Won't we have any issue if the field remains null?